content/launch-library/launch.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: launch
+title: launch api
+url: guides/launch-library/launch
```

Comment:
The content itself is not in `guides/` any longer, right? Is intentional to manually add it to the URLs? I believe if you don't want `guides/` to be part of the URL, you should not set the `url` in the page front matter.

Reply:
OK. So I think I understand why I was confused. 

The Launch docs (that are hidden) are here: https://docs.wandb.ai/guides/launch/ (you can't access them unless you know the path).

Those docs (listed in `content/launch/`) do contain the `url` key.

In my case, if I want them to show up in https://docs.wandb.ai/launch-library, change the value for the `url`  key to just `launch-library`

Reply:
(which I just did in the latest commit)

Reply:
It seems, at least based on what was done in the past, that we can set the url key in the frontmatter.